src/index.rs

```diff
@@ -76,6 +76,42 @@ impl fmt::Display for Column {
     }
 }
 
+/// Window Dimensions
+///
+/// Newtype to avoid passing values incorrectly
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Serialize, Deserialize)]
```

Comment:
I think most of these implementations are unused. Maybe just `Debug`, `Copy`, and `Clone` would suffice.